spec/jobs/scheduled/monitor_event_dates_spec.rb

```diff
@@ -97,4 +97,34 @@
       expect(events).to include(event_name: :discourse_post_event_event_ended, params: [past_event])
     end
   end
+
+  context '#due_reminders' do
```

Comment:
this test is half good, as you could just have the fact that 1.foo doesn't make the due_reminders in range... but other alternatives are testing private method which doesn’t ensure the method will be used,